### DISCUSSION TITLE
How to ensure subsequent commands run without a pause inbetween

### DISCUSSION THREAD

**Discussion by laur89** [borgbackup/borg#7358] [Q&A] [answered]
I'm using borg with two repositories - one for virtual machines, other for documents/other data.
There are around 7 `create` commands that back up data to the latter repository.

I'd like those backups to run in as short amount of time (ie avoid in-between pauses) as possible to avoid HDD wake-ups. Currently what I do is schedule them relatively close to one-another, but use a large `--lock-wait` value with each command, eg `--lock-wait 18000`. This means if first `create` is still running when subsequent backup job is launched, it waits until the first one finishes and then almost immediately kicks the backup job off.

Is this a reasonable way to go about this?

**Comment by ThomasWaldmann** [comment, marked as answer]
Yes, guess that should work (if you need to start the commands in parallel / independent of each other).

Even simpler would be to have all the commands right after each other in one script.
